maps/submaps/engine_submaps_vr/tether/_engine_submaps.dm

```diff
@@ -0,0 +1,34 @@
+// This causes engine maps to get 'checked' and compiled, when undergoing a unit test.
+// This is so Travis can validate PoIs, and ensure future changes don't break PoIs, as PoIs are loaded at runtime and the compiler can't catch errors.
+// When adding a new engine, please add it to this list.
+#if MAP_TEST
+#include "engine_rust.dmm"
+#include "engine_singulo.dmm"
+#include "engine_sme.dmm"
```

Comment:
Not sure why tesla wasn't on this list.  Maybe it should be added?

Reply:
No idea, that's how it was beforehand.